includes/feeds.php

```diff
@@ -94,8 +94,8 @@ function georss_namespace() {
 		if ( $wpgeo->show_maps() ) {
 
 			$geo_namespaces = array(
-				'georss' => 'http://www.georss.org/georss',
-				'geo'    => 'http://www.w3.org/2003/01/geo/wgs84_pos#',
+				'georss' => 'https://www.georss.org/georss',

```

Comment:
https://www.georss.org/georss is not available over https - think this should revert to http